### PR TITLE
Merge/release develop

### PR DESCRIPTION
**Pull request by ryma2fhir**
merge the release branch into develop after changing the HL7 email address and removing the NHS email address.

### CODE REVIEW

**Comment by declankieran-nhsd** [comment]
@ryma2fhir Did you get the descriptions from develop?  There were also some other conflicts on files that were pretty printed that need things lifted from both versions

File: codesystems/CodeSystem-UKCore-PersonMaritalStatusEngland.xml [hunk 1]
```diff
@@ -5,27 +5,18 @@
 	<name value="UKCorePersonMaritalStatusEngland"/>
 	<title value="UK Core Person Marital Status England"/>
 	<status value="active"/>
-	<date value="2022-12-16"/>
+	<date value="2022-08-26"/>
```
Comment (APageNHS) [inline codesystems/CodeSystem-UKCore-PersonMaritalStatusEngland.xml:8]: This files date and description will get reset to 1.0.0  -  can we exclude this one and do it manually or keep the develop branch date and desc?

File: structuredefinitions/Extension-UKCore-DeliveryChannel.xml [hunk 2]
```diff
@@ -1,74 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-	<id value="Extension-UKCore-DeliveryChannel"/>
-	<url value="https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-DeliveryChannel"/>
-	<version value="1.2.0"/>
-	<name value="ExtensionUKCoreDeliveryChannel"/>
-	<title value="Extension UK Core Delivery Channel"/>
-	<status value="active"/>
-	<date value="2022-12-16"/>
```
Comment (APageNHS) [inline structuredefinitions/Extension-UKCore-DeliveryChannel.xml:9]: Does anyone know why the develop version had an updated date/version, content looks identical at first glance

**Comment by ryma2fhir** [comment]
The list of conflcted files
 both modified:   codesystems/CodeSystem-UKCore-MedicationRequestCategory.xml
        both modified:   codesystems/CodeSystem-UKCore-NHSNumberVerificationStatusEngland.xml
        both modified:   codesystems/CodeSystem-UKCore-PersonMaritalStatusEngland.xml
        both added:      examples/UKCore-Condition-Sn-Extension-CodingSCT-CodeUnknown-Example.xml
        both modified:   examples/UKCore-DocumentReference-CarePlanReportPDF-Example.xml
        both modified:   examples/UKCore-EpisodeOfCare-SmokingCessationTherapy-Example.xml
        both modified:   examples/UKCore-MedicationStatement-Amoxicillin-Example.xml
        both modified:   examples/UKCore-MedicationStatement-Extension-MedicationPrescribingOrganizationType-Example.xml
        both modified:   examples/UKCore-MedicationStatement-Extension-MedicationStatementLastIssueDate-Example.xml
        both modified:   examples/UKCore-OperationOutcome-DateError-Example.xml
        both modified:   structuredefinitions/Extension-UKCore-ContactRank.xml
        both modified:   structuredefinitions/Extension-UKCore-DeliveryChannel.xml
        both modified:   structuredefinitions/Extension-UKCore-RelatedClinicalContent.xml
        both modified:   structuredefinitions/UKCore-AllergyIntolerance.xml
        both modified:   structuredefinitions/UKCore-Appointment.xml
        both modified:   structuredefinitions/UKCore-AuditEvent.xml
        both modified:   structuredefinitions/UKCore-Bundle.xml
        both modified:   structuredefinitions/UKCore-CarePlan.xml
        both modified:   structuredefinitions/UKCore-CareTeam.xml
        both modified:   structuredefinitions/UKCore-Communication.xml
        both modified:   structuredefinitions/UKCore-Composition.xml
        both modified:   structuredefinitions/UKCore-Condition.xml
        both modified:   structuredefinitions/UKCore-Consent.xml
        both modified:   structuredefinitions/UKCore-Device.xml
        both modified:   structuredefinitions/UKCore-DiagnosticReport.xml
        both modified:   structuredefinitions/UKCore-DocumentReference.xml
        both modified:   structuredefinitions/UKCore-Encounter.xml
        both modified:   structuredefinitions/UKCore-EpisodeOfCare.xml
        both modified:   structuredefinitions/UKCore-Flag.xml
        both modified:   structuredefinitions/UKCore-HealthcareService.xml
        both modified:   structuredefinitions/UKCore-Immunization.xml
        both modified:   structuredefinitions/UKCore-List.xml
        both modified:   structuredefinitions/UKCore-Location.xml
        both modified:   structuredefinitions/UKCore-Medication.xml
        both modified:   structuredefinitions/UKCore-MedicationAdministration.xml
        both modified:   structuredefinitions/UKCore-MedicationDispense.xml
        both modified:   structuredefinitions/UKCore-MedicationRequest.xml
        both modified:   structuredefinitions/UKCore-MedicationStatement.xml
        both modified:   structuredefinitions/UKCore-MessageHeader.xml
        both modified:   structuredefinitions/UKCore-Observation.xml
        both modified:   structuredefinitions/UKCore-OperationOutcome.xml
        both modified:   structuredefinitions/UKCore-Organization.xml
        both modified:   structuredefinitions/UKCore-Patient.xml
        both modified:   structuredefinitions/UKCore-Practitioner.xml
        both modified:   structuredefinitions/UKCore-PractitionerRole.xml
        both modified:   structuredefinitions/UKCore-Procedure.xml
        both modified:   structuredefinitions/UKCore-Provenance.xml
        both modified:   structuredefinitions/UKCore-Questionnaire.xml
        both modified:   structuredefinitions/UKCore-QuestionnaireResponse.xml
        both modified:   structuredefinitions/UKCore-RelatedPerson.xml
        both modified:   structuredefinitions/UKCore-Schedule.xml
        both modified:   structuredefinitions/UKCore-ServiceRequest.xml
        both modified:   structuredefinitions/UKCore-Slot.xml
        both modified:   structuredefinitions/UKCore-Specimen.xml
        both modified:   structuredefinitions/UKCore-Task.xml
        both modified:   valuesets/ValueSet-UKCore-AllergyManifestation.xml
        both modified:   valuesets/ValueSet-UKCore-ConditionEpisodicity.xml
        both modified:   valuesets/ValueSet-UKCore-ImmunizationExplanationReason.xml
        both modified:   valuesets/ValueSet-UKCore-ListWarningCode.xml
        both modified:   valuesets/ValueSet-UKCore-MedicationStatementCategory.xml
        both modified:   valuesets/ValueSet-UKCore-MessageEvent.xml
        both modified:   valuesets/ValueSet-UKCore-PersonMaritalStatusCode.xml


I think it'll probably be best to make the changes as we did before than try and work out what should and shouldn't be in this merge

**Comment by declankieran-nhsd** [comment]
I've nearly got it done here, I think doing the merge is better as doing it separately means there could be small differences, I'll push this soon and tag you and adam

**Comment by APageNHS** [comment]
> The list of conflcted files @declankieran-nhsd @APageNHS 
> SNIP!
> I think it'll probably be best to make the changes as we did before than try and work out what should and shouldn't be in this merge

all the structuredefintion ones will be version, date, and description, i'll double check the examples and valuesets, but agree, its probably easier to fix the emails on a develop based feature branch

**Comment by declankieran-nhsd** [comment]
Although could do both and then diff or merge from there...

**Comment by ryma2fhir** [comment]
The other way is clicking viewed for the ones that only have email changes, and then go through the ones left monday

**Comment by declankieran-nhsd** [comment]
@ryma2fhir that merge I did there didn't work, going to need to redo it.  Probably will be monday before I get it done.  I'd still prefer to do this via a merge, but there are a lot of conflicts with the descriptions and formatting meaning that there are a lot of manual decisions needing to be made.  Might even be good to do it over a screenshare so that any fiddly ones are spotted when resolving the conflicts...

**Comment by ryma2fhir** [comment]
Certainly. In the meantime I've reviewed 140+ of the files (so halfway). 10 examples need a look at. I think all profiles will also need their description changing, but that will be about it apart from the odd one or two. My eyes are going blurry so I'll pick it up monday as you say and we'll have a team meeting to go through them.

**Comment by declankieran-nhsd** [comment]
And this is the diff of my merge from develop, which should only have email and contact changes in there

https://github.com/NHSDigital/FHIR-R4-UKCORE-STAGING-MAIN/compare/develop..fix/merge-release-1.0.1

Quick eyeball of it and it looks ok to me, the only one that need a good looking at is valuesets/ValueSet-UKCore-PersonMaritalStatusCode.xml

**Pull request closed without merging**